src/content/en/2020/markup.md

```diff
@@ -579,7 +559,7 @@ In our mobile dataset of 6.3 million pages, around 0.9 million pages (14.01%) co
     <thead>
       <tr>
         <th>Element</th>
-        <th>Occurances</th>
+        <th>Pages</th>
```

Comment:
See above (#467)—here, too, this would be wrong. This information does *not* relate to pages.

We should fix “Occurances” though—strangely, however, I can’t spot this misspelled variant in the live version of the chapter.

Reply:
@Tiggerito can you confirm?

Looking at [this part of the query](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/03_Markup/pages_element_count_by_device_and_obsolete_elements.sql):

```sql
  COUNT(0) AS freq,
  AS_PERCENT(COUNT(0), total) AS pct_pages_with_obsolete_elements,
  AS_PERCENT(COUNT(0), SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX)) AS ratio_compared_to_all_obsolete_elements
```

If we're dividing by total (which is based on pages) as in the second line then surely `COUNT(0)` must be pages too?

Reply:
Oh and btw looks like I introduced the misspelling of Occurances in #1857 when converting from Markdown tables to HTML (to allow correct alignment of numeric values). I deliberately did that separately to this PR to keep this PR on the edits.

Reply:
Yes, I believe this one is pages based. It's only looking to see if an obsolete element exists on a page, nut how many times it is there.